patterns/2-structured/communication-tooling.md

```diff
@@ -77,6 +77,7 @@ to a lower need to repeat explanations.
 * Europace AG
 * Paypal Inc.
 * Mercado Libre
+* Siemens is heavily using communication channels accessible for every employee for all user questions and discussions around code.siemens.com, Inner Source and Open Source.
```

Comment:
Maybe we could rephrase to avoid the double `for`?

```suggestion
* Siemens makes extensive use of communication channels accessible to all employees for all user questions and discussions around code.siemens.com, Inner Source and Open Source.
```

Reply:
That's why I love your reviews :)